src/mixin-tracking.ts

```diff
@@ -10,7 +10,7 @@ export const getMixinsForClass = (clazz: Class) =>
 export const registerMixins = (mixedClass: any, constituents: Function[]) =>
 	mixins.set(mixedClass, constituents);
 
-export const hasMixin = <M>(instance: any, mixin: new (...args) => M): instance is M => {
+export const hasMixin = <M>(instance: any, mixin: (new (...args) => M) | (abstract new (...args) => M)): instance is M => {
```

Comment:
`abstract new (...args) => M` accepts both abstract and concrete classes, so let's just use the right hand side of this union instead

Reply:
This is how it's done for [the `Class` type](https://github.com/tannerntannern/ts-mixer/blob/master/src/types.ts#L51) too

Reply:
Ahh, good to know, thanks! Fixed.